lib/modules/fuzzer/index.js

```diff
@@ -17,14 +17,18 @@ class ContractFuzzer {
   generateFuzz(iterations, contract) {
     const self = this;
     let fuzzMap = {};
-    for (let i = 0; i < iterations; i++) {
-      contract.abiDefinition.filter((x) => x.inputs && x.inputs.length != 0).forEach((abiMethod) => {
-        let name = abiMethod.type === "constructor" ? "constructor" : abiMethod.name;
-        let inputTypes = abiMethod.inputs.map(input => input.type);
-        fuzzMap[name] = inputTypes.map(input => this.getTypeFuzz(input));
-      });
-    }
-    for (let key in fuzzMap) self.logger.info(key + ":" + fuzzMap[key]);
+    contract.abiDefinition.filter((x) => x.inputs && x.inputs.length != 0).forEach((abiMethod) => {
+      let name = abiMethod.type === "constructor" ? "constructor" : abiMethod.name;
+      let inputTypes = abiMethod.inputs.map(input => input.type);
+      fuzzMap[name] = {};
+      for (let i = 0; i < iterations; i++) {
+        fuzzMap[name][i] = {};
+        fuzzMap[name][i] = inputTypes.map(input => this.getTypeFuzz(input));
+      }
+      self.logger.info(name);
+      for (let key in fuzzMap[name]) self.logger.info("iteration: " + key + "\n" + fuzzMap[name][key]);
```

Comment:
I just saw, you can actually do that log inside the previous loop, just after setting`fuzzMap[name][i]`